spec/requests/status_spec.rb

```diff
@@ -0,0 +1,111 @@
+# frozen_string_literal: true
+
+describe '/status.json' do
+  before do
+    BawWorkers::Config.original_audio_helper.possible_dirs.each { |path|
+      FileUtils.mkdir_p(path)
+    }
+  end
+
+  example 'everything ok' do
+    get '/status.json'
```

Comment:
We don't have a test for `/status`, or is it an assumption that it'll work because the `/status.json` version works?

Reply:
Ahhh, I just saw in status_routing_spec you test that both return the same thing